src/vaadin-pro-grid-inline-editing-mixin.html

```diff
@@ -77,6 +80,14 @@
       this.notifyPath('items.' + index + '.' + path, value);
     }
 
+    _isAlphaNumericKey(keyCode) {
+      return (
+        (keyCode >= 65 && keyCode <= 90) || // ascii a-z
```

Comment:
English only? What about other languages?

Reply:
I have copied the code from `iron-a11y-keys-behavior`:
https://github.com/PolymerElements/iron-a11y-keys-behavior/blob/26910e31ec4057150dd6f97e3818385638212a43/iron-a11y-keys-behavior.js#L146

Will look if there is a more comprehensive way to approach this.

Reply:
Updated to check for `e.key.length === 1` to match any printable character, in order to not dive really deep into the world of fancy ES2015 Unicode RegExp.

Also, this is how Google Spreadsheet works, and that's where we have been told to take inspiration from 😄

Reply:
> Updated to check for e.key.length === 1 to match any printable character

FFS!